react-facet/docs/getting-started.md

```diff
@@ -31,6 +31,22 @@ const Counter = () => {
 render(<Counter />, document.getElementById('root'))
 ```
 
+## Installation
+
+The packages of React Facet are grouped under the `@react-facet` scope on npm. To install the packages for the example above, run:
+
+```sh
+yarn add @react-facet/core @react-facet/dom-fiber
+```
+
+Note that if you want to use the [DOM Components](api/dom-components) you will need to install those separately:
```

Comment:
How about deleting the paragraph " …and so on for any other required package." and instead changing the earlier paragraph to this:

```suggestion
Note that if you want to use other [packages](api/packages), such as the [DOM Components](api/dom-components), you will need to install them separately:
```